src/System.Linq.Parallel/src/System/Linq/Parallel/Channels/SynchronousChannel.cs

```diff
@@ -29,15 +29,7 @@ internal sealed class SynchronousChannel<T>
 #if DEBUG
         // In debug builds, we keep track of when the producer is done (for asserts).
         private bool _done;
-#endif
-
-        //-----------------------------------------------------------------------------------
-        // Instantiates a new queue.
-        //
-
-        internal SynchronousChannel()
```

Comment:
Absolutely you're right, yet I don't think it's gonna be a problem. Do you have any scenario that can cause an error?